lib/public/views/Logs/Logs.js

```diff
@@ -24,18 +24,18 @@ export default class Overview extends Observable {
     constructor(model) {
         super();
         this.model = model;
+
         this.filterCriteria = [];
-        this.data = [];
-        this.filteredData = [];
+        this.logs = RemoteData.NotAsked();
```

Comment:
You can replace this with the `clearLogs` method

Reply:
Done